--- wand-4.3/device/fsl/wandboard/init.rc.orig	2014-01-08 07:23:01.000000000 +0100
+++ wand-4.3/device/fsl/wandboard/init.rc	2014-02-17 18:33:30.525340616 +0100
@@ -418,6 +418,9 @@
 #  Set GPU 3D minimum clock to 3/64
    write /sys/module/galcore/parameters/gpu3DMinClock 3
 
+# Enable adb over network
+    setprop service.adb.tcp.port 5555
+
 # set this to not let jni check crash, just warnning message.
     setprop dalvik.vm.jniopts warnonly
 
@@ -514,6 +517,10 @@
 on property:ro.kernel.qemu=1
     start adbd
 
+# always start adbd
+on property:ro.secure=0
+    start adbd
+
 service servicemanager /system/bin/servicemanager
     class core
     user system
